Sources/Datadog/RUM/RUMMonitor/Scopes/RUMViewScope.swift

```diff
@@ -421,7 +421,14 @@ internal class RUMViewScope: RUMScope, RUMContextProvider {
 
     private func sendViewUpdateEvent(on command: RUMCommand, context: DatadogContext, writer: Writer) {
         version += 1
-        attributes.merge(rumCommandAttributes: command.attributes)
+
+        // RUMM-3133 Don't override View attributes with UserAction, Resource or LongTask attributes
+        switch command {
+        case is RUMUserActionCommand, is RUMResourceCommand, is RUMAddLongTaskCommand:
+            break
+        default:
+            attributes.merge(rumCommandAttributes: command.attributes)
+        }
```

Comment:
I think we should do the opposite and check for view-related command types instead.
Maybe it could use a
```swift
internal protocol RUMViewCommand: RUMCommand { }
```
for selecting which command types can propagate attributes to the view-scope. It would be less error prone when we add command  types. WDYT?

Reply:
For sure having one base protocol for all the commands that we don't want to propagate would be ideal. I like the idea but I'm not sure what other commands we can expect here. This is the same rational behind using `break` with current `default` behavior.

Any hints on commands that we don't want propagation for?